README.rst

```diff
@@ -2,55 +2,132 @@
+   :alt: Github Actions status
    :target: https://github.com/shapely/shapely/actions?query=branch%3Amain
 
+.. Travis CI status -- https://travis-ci.com
+
+.. image:: https://travis-ci.com/shapely/shapely.svg?branch=main
+   :alt: Travis CI status
+   :target: https://travis-ci.com/github/shapely/shapely
+
+.. PyPI
+
+.. image:: https://img.shields.io/pypi/v/shapely.svg
+   :alt: PyPI
+   :target: https://pypi.org/project/shapely/
+
+.. Anaconda
+
+.. image:: https://img.shields.io/conda/vn/conda-forge/shapely
+   :alt: Anaconda
+   :target: https://anaconda.org/conda-forge/shapely
+
+.. Coverage
+
 .. |coveralls| image:: https://coveralls.io/repos/github/shapely/shapely/badge.svg?branch=main
    :target: https://coveralls.io/github/shapely/shapely?branch=main
 
+.. Zenodo
+
+.. .. image:: https://zenodo.org/badge/191151963.svg
+..   :alt: Zenodo
+..   :target: https://zenodo.org/badge/latestdoi/191151963
+
 Manipulation and analysis of geometric objects in the Cartesian plane.
 
 .. image:: https://c2.staticflickr.com/6/5560/31301790086_b3472ea4e9_c.jpg
    :width: 800
    :height: 378
 
 Shapely is a BSD-licensed Python package for manipulation and analysis of
-planar geometric objects. It is based on the widely deployed `GEOS
-<https://libgeos.org/>`__ (the engine of `PostGIS
-<https://postgis.net/>`__) and `JTS
-<https://locationtech.github.io/jts/>`__ (from which GEOS is ported)
-libraries. Shapely is not concerned with data formats or coordinate systems,
-but can be readily integrated with packages that are. For more details, see:
-
-* `Shapely GitHub repository <https://github.com/shapely/shapely>`__
-* `Shapely documentation and manual <https://shapely.readthedocs.io/en/latest/>`__
+planar geometric objects. It is using the widely deployed open-source
+geometry library `GEOS <https://libgeos.org/>`__ (the engine of `PostGIS
+<https://postgis.net/>`__, and a port of `JTS <https://locationtech.github.io/jts/>`__).
+Shapely wraps the GEOS operations in NumPy ufuncs providing a performance
+improvement when operating on arrays of geometries, as well as provides a
+rich scalar `Geometry` interface.
+Shapely is not concerned with data formats or coordinate systems, but can be
+readily integrated with packages that are.
+
+What is a ufunc?
+----------------
+
+A universal function (or ufunc for short) is a function that operates on
+n-dimensional arrays in an element-by-element fashion, supporting array
+broadcasting. The for-loops that are involved are fully implemented in C
+diminishing the overhead of the Python interpreter.
+
+Multithreading
+--------------
+
+PyGEOS functions support multithreading. More specifically, the Global
+Interpreter Lock (GIL) is released during function execution. Normally in Python, the
+GIL prevents multiple threads from computing at the same time. PyGEOS functions
+internally releases this constraint so that the heavy lifting done by GEOS can be
+done in parallel, from a single Python process.
```

Comment:
For Shapely, those two paragraphs can probably be moved to the actual documentation instead of being in the README (but that can be done later when reworking the docs, currently the README gets inserted in the docs anyway)